bin/strata-cli/src/main.rs

```diff
@@ -10,9 +10,9 @@ pub mod strata;
 pub mod taproot;
 
 use cmd::{
-    backup::backup, balance::balance, change_pwd::change_pwd, config::config, drain::drain,
-    faucet::faucet, receive::receive, recover::recover, reset::reset, scan::scan, send::send,
-    withdraw::withdraw, Commands, TopLevel,
+    backup::backup, balance::balance, change_pwd::change_pwd, config::config, deposit::deposit,
+    drain::drain, faucet::faucet, receive::receive, recover::recover, reset::reset, scan::scan,
+    send::send, withdraw::withdraw, Commands, TopLevel,
 };
```

Comment:
This isn't really related but could we reexport all of these impl fns into the mod.rs so we can import them all here as `use cmd::*;`?